### PR TITLE
WIP: Setup website

### PR DESCRIPTION
**Pull request by trabaldini**
Used the color of the logo for background and links, but maybe it's a bit too much?
Used the content of the README, but it's not so much. Is there other content I could/should add?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1 +1,402 @@
...
+    </div>
+    <script src="https://yandex.st/highlightjs/7.4/highlight.min.js"></script>
+    <script type="text/javascript">
+      hljs.initHighlightingOnLoad();
+      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+      ga('create', 'UA-53237918-6', 'auto');
```
Comment (marcoow) [inline index.html]: I just created the correct analytics ID: `UA-53237918-22`.

File: index.html [hunk 2]
```diff
@@ -1 +1,402 @@
...
+              button.on('click', function() {
+                document.cookie = "cookiesAccepted=true;path=/;expires=Mon, 24 Mar 2319 09:52:10 GMT";
+                banner.remove();
+              });
+            });
+          </script>
+        </div>
+      </div>
+    </div>
+    <script src="https://yandex.st/highlightjs/7.4/highlight.min.js"></script>
```
Comment (marcoow) [inline index.html]: I don't think this is needed.

File: index.html [hunk 3]
```diff
@@ -1 +1,402 @@
...
+      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+      ga('create', 'UA-53237918-6', 'auto');
+      ga('send', 'pageview');
+    </script>
+  </body>
+</html>
```
Comment (marcoow) [inline index.html]: 👀

File: index.html [hunk 4]
```diff
@@ -1 +1,402 @@
...
+    <![endif]-->
+    <style type="text/css" media="screen">
+      body {
+        line-height: 1.7;
+        font-weight: 300;
+        color: #384147;
+      }
+    
+      a, a:visited {
+        color: #99FFCC;
```
Comment (marcoow) [inline index.html]: the link text is a bit hard to read in this color - we might have to add a background color, e.g. black.

File: index.html [hunk 5]
```diff
@@ -1 +1,402 @@
...
+        margin-right: 10%;
+        margin-left: 10%;
+      }
+
+      #intro i.main-logo {
+        margin: 130px 0 58px 0;
+      }
+
+      hr {
+        border-bottom: 1px solid #99FFCC;
```
Comment (marcoow) [inline index.html]: I think this might look better in black.

**Comment by trabaldini** [comment]
Changed the colors and removed the `.DS_Store`, but I can't find the `.travis.yml`
There is just one on the master branch but it's not the one you mean?

**Comment by marcoow** [comment]
Hm, I assumed the `.travis.yml` was still present and thus Travis tried to build. I disabled the build for this branch in the Travis settings now.

**Pull request merged**